<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="L-SFH79FJG5J-M" failures="1" tests="1" name="SFDCLogin_TC1_4.Login" time="6.713" errors="0" timestamp="2023-02-18T11:56:13 PST" skipped="0">
  <testcase classname="SFDCLogin_TC1_4.Login" name="loginSF_TC_2" time="6.713">
    <failure message="expected [https://tekarch70-dev-ed.develop.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom] but found [https://tekarch70-dev-ed.develop.my.salesforce.com/_ui/identity/verification/method/EmailVerificationFinishUi/e?vcsrf=Bt5JWH8Ep8TPj5Qd_tUbyyFJqcRiItef_I38p3TqtMVfxw8eib1hLQRbsRp1AZfB1HD3CAMygiM6WALab8dRpDKGQbS5TU7VsUC1EEw4IsvwdC45NHb1vaGRne87qhEnv1HUT9Ti8gAfL9CWf-bzPf6PmFrqPMH5XSa5PazBk1efpyktDaTg35LGQgn31sS7kWMnhHWWAx-TK7SpxrnM4reqfXMKkq3xvPF8Yr87xei1jipyfbh-oO7aAJtDUFY9MdyBdb3-N-lMVvBUsqVH5Hiy_JLBZcEu7JQu7B-zDCb9wN0qyp_ingRhCk6IUPF_ea6vxsdYXCGyN5fAFxlShiLXZH_3LTjqLtzwQDEHHI7niXRZWv76c0M80y0JjqTdYcOYfwmKsk6hhHDAnjs2m-hUEqJkq2TKXKL52_sOzsBziYGk6GsyIflIGQnJhw0TeSmhlmVjZvBSEnQSEwkKLXyPcAktLnTBzP3Am6eHCjg2IEbB9BSrEgZ1to9EIqheaBMpfT9rHaxfXBKbrV8FYDWz-4mAphqjETKa0rJrC8n2iC_YcfF9QM2aK7_HTyb6vMdRY-P9axNojNU6ot5lSV6DKjRaDM0HFkamhmvkyp6naPkdzW_pm71A-aW6Yqks7HNg3hTVsdG_JvQUp6Mk6t8DZNw23hMdavAVLjzRkoI%3D&amp;vpol=ic&amp;vflid=0&amp;vfgrp=1684842964&amp;retURL=%2Fsecur%2Ffrontdoor.jsp%3Fallp%3D1%26cshc%3Dn000004nUvnn00000ACpTC%26apv%3D1%26display%3Dpage%26ucs%3D1]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [https://tekarch70-dev-ed.develop.my.salesforce.com/setup/forcecomHomepage.apexp?setupid=ForceCom] but found [https://tekarch70-dev-ed.develop.my.salesforce.com/_ui/identity/verification/method/EmailVerificationFinishUi/e?vcsrf=Bt5JWH8Ep8TPj5Qd_tUbyyFJqcRiItef_I38p3TqtMVfxw8eib1hLQRbsRp1AZfB1HD3CAMygiM6WALab8dRpDKGQbS5TU7VsUC1EEw4IsvwdC45NHb1vaGRne87qhEnv1HUT9Ti8gAfL9CWf-bzPf6PmFrqPMH5XSa5PazBk1efpyktDaTg35LGQgn31sS7kWMnhHWWAx-TK7SpxrnM4reqfXMKkq3xvPF8Yr87xei1jipyfbh-oO7aAJtDUFY9MdyBdb3-N-lMVvBUsqVH5Hiy_JLBZcEu7JQu7B-zDCb9wN0qyp_ingRhCk6IUPF_ea6vxsdYXCGyN5fAFxlShiLXZH_3LTjqLtzwQDEHHI7niXRZWv76c0M80y0JjqTdYcOYfwmKsk6hhHDAnjs2m-hUEqJkq2TKXKL52_sOzsBziYGk6GsyIflIGQnJhw0TeSmhlmVjZvBSEnQSEwkKLXyPcAktLnTBzP3Am6eHCjg2IEbB9BSrEgZ1to9EIqheaBMpfT9rHaxfXBKbrV8FYDWz-4mAphqjETKa0rJrC8n2iC_YcfF9QM2aK7_HTyb6vMdRY-P9axNojNU6ot5lSV6DKjRaDM0HFkamhmvkyp6naPkdzW_pm71A-aW6Yqks7HNg3hTVsdG_JvQUp6Mk6t8DZNw23hMdavAVLjzRkoI%3D&vpol=ic&vflid=0&vfgrp=1684842964&retURL=%2Fsecur%2Ffrontdoor.jsp%3Fallp%3D1%26cshc%3Dn000004nUvnn00000ACpTC%26apv%3D1%26display%3Dpage%26ucs%3D1]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Reusable.BaseClass.validation(BaseClass.java:94)
at SFDCLogin_TC1_4.Login.loginSF_TC_2(Login.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginSF_TC_2 -->
  <system-out/>
</testsuite> <!-- SFDCLogin_TC1_4.Login -->
